### PR TITLE
CSS add transition

### PR DESCRIPTION
**Pull request by reeceatkinson**
### Description
Edited the CSS code to transition aswell as animation-duration.

### Type of Change
- Updating the documentation

### Checklist
- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Comment by Dusch4593** [comment]
@reeceatkinson Thanks for this! I'm gonna take a look in a sec and review it. 😄

**Comment by reeceatkinson** [comment]
@Dusch4593 Thanks!

File: content/css/concepts/animations/animations.md [hunk 1]
```diff
@@ -38,5 +38,6 @@ h1 {
   color: yellow;
   animation-name: color-change;
   animation-duration: 5s;
+  transition: 5s;
 }
```
Comment (Dusch4593) [inline content/css/concepts/animations/animations.md:42]:
```suggestion
/* The animation */
@keyframes color-change {
  from {
    color: purple;
  }
  25% {
    color: yellow;
  }
  50% {
    color: red;
  }
  75% {
    color: blue;
  }
  to {
    color: yellow:
  }
}

/* The element */
h1 {
  color: purple;
  animation-name: color-change;
  animation-duration: 5s;
}
```

We should edit the `@keyframes` example to highlight how CSS animations allow multiple style changes between frames.

File: content/css/concepts/animations/animations.md [hunk 2]
```diff
@@ -38,5 +38,6 @@ h1 {
   color: yellow;
   animation-name: color-change;
   animation-duration: 5s;
+  transition: 5s;
```
Comment (Dusch4593) [inline content/css/concepts/animations/animations.md:41]:
```suggestion
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
Hi @reeceatkinson Thank you again for submitting this PR. I just took at look at your changes and am sorry to say I don't think they don't quite fit with the purpose of this particular entry. 

This example is using terms and concepts related to [CSS Animations](https://www.codecademy.com/resources/docs/css/animations). Your change deals with a different set of animation styles called [CSS transitions](https://www.codecademy.com/resources/docs/css/transition).

However, there is still something to be gained from this PR! 😃 I think this entry could still use some edits, so I left some comments/suggestions for you to look over and resolve.

Tag me when you're finished pushing more changes and I'll take another look. Thanks again, @reeceatkinson!

**Comment by sonnynomnom** [comment]
Closing this one for now. Feel free to make another entry @reeceatkinson based on what Brandon suggested!

**Pull request closed without merging**